source/crud/tests/write/bulkWrite-arrayFilters.yml

```diff
@@ -12,21 +12,23 @@ tests:
             arguments:
                 requests:
                     -
+                        description: "UpdateOne when one document matches arrayFilters"
```

Comment:
AFAIK `description` is only a top-level test field. If you'd just like to note that we're copying another test here I think a YAML comment would be preferable.

Reply:
Done and merging, thanks!